database/src/main/java/org/mifos/chatbot/database/dao/UserRepository.java

```diff
@@ -0,0 +1,149 @@
+package org.mifos.chatbot.database.dao;
+
+import org.mifos.chatbot.database.model.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class UserRepository {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public List<User> findAll() {
+        return jdbcTemplate.query(
+                "SELECT * FROM users_creds",
+                (rs, rowNum) -> new User(rs.getString("Username"),
+                        rs.getString("secret_Pass"), rs.getString("FB_userID"), rs.getString("slack_userID"), rs.getString("telegram_userID"), rs.getString("skype_userID"))
+        );
+    }
+
+    public User findUserByUsername(String username) {
+        List<User> l = jdbcTemplate.query(
+                "SELECT * FROM users_creds WHERE Username = '" + username + "'",
+                (rs, rowNum) -> new User(
+                        rs.getString("Username"),
+                        rs.getString("secret_Pass"),
+                        rs.getString("FB_userID"),
+                        rs.getString("slack_userID"),
+                        rs.getString("telegram_userID"),
+                        rs.getString("skype_userID"))
+        );
+        try {
+            return l.get(0);
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    public User findUserByFBID(String FB_userID) {
+        List<User> l = jdbcTemplate.query(
+                "SELECT * FROM users_creds WHERE FB_userID = '" + FB_userID + "'",
+                (rs, rowNum) -> new User(
+                        rs.getString("Username"),
+                        rs.getString("secret_Pass"),
+                        rs.getString("FB_userID"),
+                        rs.getString("slack_userID"),
+                        rs.getString("telegram_userID"),
+                        rs.getString("skype_userID"))
+        );
+        try {
+            return l.get(0);
+        } catch (Exception e) {
```

Comment:
Appropriately log the error so we can get back to it later if needed

Reply:
Added appropriate log message